package.json

```diff
@@ -67,7 +74,7 @@
   },
   "files": [
     "out/src",
-    "out/third_party/types",
+    "out/third_party/cloud-debug-nodejs",
```

Comment:
What is this for?

Reply:
scanner.ts from the debug agent is copied into (and not modified) ts/third_party/cloud-debug-nodejs